src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md

```diff
@@ -0,0 +1,314 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: We will look at what variable fonts are, how we can use them in our work.
+
+{# wf_updated_on: 2018-02-19 #}
+{# wf_published_on: 2018-02-19 #}
+
+# Introduction to variable fonts on the web {: .page-title }
+
+{% include "web/_shared/contributors/mustafa.html" %}
+
+In this article, we will look at what variable fonts are, how we can use them in
+our work, and the potential possibilities they entail. But to understand what
+they offer, first, we must explore how typography and font loading currently
+work on the web.
+
+## Introduction
+
+The terms font and typeface are used interchangeably in the developer industry.
+However there is a difference: A typeface includes the entire family design, for
+example, [Roboto](https://fonts.google.com/specimen/Roboto), while a font is one
+of the digital files of that family, like Roboto Bold or Roboto Italic. In other
+words, a typeface is what you see, and the font is what you use.
+
+
+<div class="attempt-left">
+  <figure>
+    <img src="images/robot-specimen.png" alt="A specimen of Roboto">
+  </figure>
+</div>
+<div class="attempt-right">
+  <figure>
+    <img src="images/roboto-family.png" alt="Different members of the Roboto family">
+  </figure>
+</div>
+<div class="clearfix"></div>
+
+<div class="note"><strong>Note:</strong><span> Roboto Family designed by Christian Robertson.
```

Comment:
The span isn't required here. If you wanted you could simply do:
```
Note: Roboto Family designed by Christian Robertson.
```

DevSite will automatically handle the formatting and such for you.